introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-generation-flan-t5-Batch-Transform.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
<p>Please don't forget to remove TODO.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Sure</p>